presto-main/src/test/java/io/prestosql/execution/TestCallTask.java

```diff
@@ -157,8 +186,12 @@ private Session testSession(TransactionManager transactionManager)
                 .build();
     }
 
-    public static void testingMethod()
+    public static class TestingProcedure
```

Comment:
Create separate procedure for your new test, otherwise it is hard to tell why `testExecuteNoPermission` passes